src/computed/channel.js

```diff
@@ -35,40 +35,59 @@ const ComputedChannel = store => {
       });
       return all;
     },
-    get channelBalanceOpenLabel() {
-      const { channels, settings } = store;
-      const sum = (channels || [])
+    get channelBalanceOpenSatoshis() {
```

Comment:
FWIW none of the unit tests cover this property or the other `.*Satoshi` computed channel properties. But, I don't think this is a huge problem.